mne/cov.py

```diff
@@ -1431,11 +1435,12 @@ def _get_whitener_data(info, noise_cov, picks, diag=False, rank=None,
         noise_cov = cp.deepcopy(noise_cov)
         noise_cov['data'] = np.diag(np.diag(noise_cov['data']))
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings']
```

Comment:
it's a doc issue. I'll send you a forthcoming conference paper on regularization of cov which more systematically discusses these choices. If you underscale, it can help in case you dit not regularize the cov.